debian/ckb-next-daemon.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
Is it not possible to use the cmake-generated service?
I don't think it's a good idea to maintain multiple versions

Reply:
This was on oversight on my part. Updated